book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = [] # Initialize an empty array to store rentals
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
Nice job adding a method for adding rentals😉, Kindly note that the method you currently use, adds an already existing Rental object to a collection associated with the current instance. but what we want to achieve here is to create a new Rental object and associate it with the current instance and a person😉. kindly see the suggestion below👍🏼.

```suggestion
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```